### PR TITLE
[GH-18190] Remove all instances of "zz" + model.NewId() in storetest/channel_sto…

### PR DESCRIPTION
**Pull request by yedamao**
#### Summary

Remove all instances of "zz" + model.NewId() in storetest/channel_store.go
#### Ticket Link

Fixes https://github.com/mattermost/mattermost-server/issues/18190
#### Release Note

```release-note

```

### CODE REVIEW

File: store/storetest/channel_store.go [hunk 1]
```diff
@@ -442,7 +442,7 @@ func testChannelStoreGetChannelsByIds(t *testing.T, ss store.Store) {
 	o1 := model.Channel{}
 	o1.TeamId = model.NewId()
 	o1.DisplayName = "Name"
-	o1.Name = "aa" + model.NewId() + "b"
+	o1.Name = "aa" + model.NewId()
```
Comment (isacikgoz) [inline store/storetest/channel_store.go:445]: Are these remaining prefixes intentional?

Comment (yedamao) [reply store/storetest/channel_store.go:445]: yes. remove the prefix tests will not pass

**Review by agnivade — APPROVED** [review]

**Comment by yedamao** [comment]
@isacikgoz `SqlChannelStore.GetChannelsByIds` will sort channels by Name, so these aa/bb/cc prefixes is intentional.

PTAL.

**Review by isacikgoz — APPROVED** [review]
Thanks for the heads up @yedamao, LGTM 👍

**Pull request merged**